FordLabs/retroquest
# Move cypress utils to commands file and cleanup tests

## Overview
Move cypress utils to commands file and cleanup tests. This is another PR to merge in work done on the `react-migration` branch that will help there be less merge conflicts when rebasing if it was in main.

Connects N/A

### Demo
App should work as it did before.

## Testing Instructions
* Run cypress tests and ensure they all pass (minus the two currently flaky tests)
